### ISSUE TITLE
Ver 1.1 Configuration Issue

### ISSUE THREAD

**Issue by tjreeder90** [doksu/TA-asngen#15]
I am getting the following error when running the search to populate the asn lookup following the update to ver 1.1. I checked the asngen.conf file and my license key is there, and I have added it in the setup section of the app, so I'm not sure what could be causing the issue. Is there something else I need to configure in the .conf or .py files?

Error: \TA-asngen\bin\asngen.py", line 37 : Error reading configuration. Please check your local asngen.conf file.

**Comment by doksu** [comment]
Hi @tjreeder90, is the asngen.conf file definitely in the local and not the default folder for the app on the search head?

**Comment by tjreeder90** [comment]
Hi doksu. Yes, there is one in both local and default, and the key is
definitely in the one in local. Is there something I need to do to
configure the license in my MaxMind account for GeoLite?

**Comment by doksu** [comment]
Hmm, I'm not sure why it's not working for you. Maybe they have multiple types of keys? Is this the place you got it? https://www.maxmind.com/en/geolite2/signup

**Comment by tjreeder90** [comment]
Yes that is where I signed up. The license key generator does not have any options for different types of keys other than whether it is for GeoIP Update, which is a different product. I generated a second key just to see if there was an issue with the first one and had the same result.

**Comment by reswob10** [comment]
It seems you may need the user ID as well?
